source/extensions/omni.isaac.lab/omni/isaac/lab/utils/configclass.py

```diff
@@ -98,6 +98,7 @@ class EnvCfg:
     setattr(cls, "from_dict", _update_class_from_dict)
     setattr(cls, "replace", _replace_class_with_kwargs)
     setattr(cls, "copy", _copy_class)
+    setattr(cls, "assert_valid", _assert_valid)
```

Comment:
NIT: Maybe a simpler name?

```suggestion
    setattr(cls, "validate", _validate)
```